MapboxAndroidDemo/gradlew.bat

```diff
@@ -0,0 +1,90 @@
+@if "%DEBUG%" == "" @echo off
```

Comment:
Remove this gradlew file and add to .gitignore. I assume this got included by accident since @LornTao looks to be using windows?